src/storageClients/CassandraTableStorageManager.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const Promise = require('bluebird');
+
+const BATCH_LIMIT = 10;
+
+module.exports = {
+
+  prepareInsertTopic: function(topic) { //TODO: the insert statement to reflect the final topic schema
```

Comment:
yes, that makes sense. I'm a little new to es6 arrow functions, so I think I accidentally forgot to make this into one...and possibly others. I'll change those as well if I find more.